pom.xml

```diff
@@ -0,0 +1,882 @@
+    <commons-httpclient.version>3.1</commons-httpclient.version>
+    <commons-lang-version>2.6</commons-lang-version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
+    <datastax.version>2.0.5</datastax.version>
+    <dumbster.version>1.6</dumbster.version>
+    <es.version>1.6.0</es.version>
+    <es-hadoop.version>2.1.0</es-hadoop.version>
+    <fastutil.version>6.6.1</fastutil.version>
+    <geronimo-jms.version>1.1.1</geronimo-jms.version>
+    <guava.version>13.0.1</guava.version>
+    <hadoop.version>2.3.0</hadoop.version>
+    <hsql.version>2.2.4</hsql.version>
+    <cdh.hive.version>1.1.0-cdh5.5.1</cdh.hive.version>
+    <javamail.version>1.4.1</javamail.version>
+    <junit.version>4.11</junit.version>
+    <kafka.version>0.8.2.2</kafka.version>
+    <mockftp.version>2.6</mockftp.version>
+    <hydrator.version>2.1.0</hydrator.version>
+    <snappy.version>1.1.2</snappy.version>
+    <slf4j.version>1.7.5</slf4j.version>
+    <twill.version>0.9.0</twill.version>
+    <twitter4j.version>4.0.3</twitter4j.version>
+    <zookeeper.version>3.4.5</zookeeper.version>
+    <jython.version>2.5.2</jython.version>
+    <json-path.version>2.2.0</json-path.version>
+    <json.version>20160212</json.version>
+    <netty.version>4.1.16.Final</netty.version>
+    <netty-http.version>1.1.0</netty-http.version>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>sonatype</id>
+      <url>https://oss.sonatype.org/content/groups/public</url>
+    </repository>
+    <repository>
+      <id>sonatype-snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </repository>
+  </repositories>
+
+  <pluginRepositories>
+    <pluginRepository>
+      <id>sonatype</id>
+      <url>https://oss.sonatype.org/content/groups/public/</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>${slf4j.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>jcl-over-slf4j</artifactId>
+        <version>${slf4j.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>javax.annotation</groupId>
+        <artifactId>jsr250-api</artifactId>
+        <version>1.0</version>
+      </dependency>
+      <dependency>
+        <groupId>co.cask.cdap</groupId>
+        <artifactId>cdap-api</artifactId>
+        <version>${cdap.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>co.cask.cdap</groupId>
+        <artifactId>cdap-api-spark</artifactId>
+        <version>${cdap.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>co.cask.cdap</groupId>
+        <artifactId>cdap-etl-proto</artifactId>
+        <version>${cdap.version}</version>
+        <scope>provided</scope>
```

Comment:
I believe nothing from proto should be used in the plugin source code itself, only in the unit tests. If so, this should be test scope

Reply:
done